README.md

```diff
@@ -135,15 +135,15 @@ If you think this list is incomplete, consider creating an [Issue](https://githu
 ## Citing ToQUBO.jl
 If you use `ToQUBO.jl` in your work, we kindly ask you to include the following citation:
 ```tex
-@software{toqubo:2022,
-  author       = {Pedro Xavier and Tiago Andrade and Joaquim Garcia and David Bernal},
-  title        = {{ToQUBO.jl}},
-  month        = {mar},
-  year         = {2022},
+@software{toqubo:2023,
+  author       = {Pedro Maciel Xavier and Pedro Ripper and Tiago Andrade and Joaquim Dias Garcia and David E. Bernal Neira},
+  title        = {ToQUBO.jl},
```

Comment:
Use the double {{}} as it would show up nicely in the latex files that people use.